composer.json

```diff
@@ -24,12 +24,12 @@
         "docs": "https://github.com/leanphp/phpspec-code-coverage#phpspec-code-coverage"
     },
     "require": {
-        "php": "~5.6||~7.0",
```

Comment:
I think it should be kept like this

Reply:
The PR switches to PhpSpec 4, which only works in PHP 7

Reply:
The update is correct. For anyone, who will use PHP 5.x, the composer will use the older version of this package (the one before this PR is merged).